doc/source/how-to/continuous-integration.rst

```diff
@@ -176,3 +176,356 @@ Workflow examples are provided for checking :ref:`Coding style`,
+  | **write:discussion**                                                                | |:x:|                  |
+  |  Read and write team discussions.                                                   |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **read:discussion**                                                                 | |:x:|                  |
+  |  Read team discussions.                                                             |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **admin:enterprise**                                                                | |:x:|                  |
+  |  Full control of enterprises.                                                       |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **manage_runners:enterprise**                                                       | |:x:|                  |
+  |  Manage enterprise runners and runner groups.                                       |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **manage_billing:enterprise**                                                       | |:x:|                  |
+  |  Read and write enterprise billing data.                                            |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **read:enterprise**                                                                 | |:x:|                  |
+  |  Read enterprise profile data.                                                      |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **audit_log**                                                                       | |:x:|                  |
+  |  Full control of audit log.                                                         |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **read:audit_log**                                                                  | |:x:|                  |
+  |  Read access of audit log.                                                          |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **codespace**                                                                       | |:x:|                  |
```

Comment:
So, codespace is lower case here in this section, if this influences your response to the capitalization question posed earlier?

Reply:
I need to update this yes.

Reply:
I'm going to go with "C"odespace(s)

Reply:
@germa89 Virtually all stye guides say not to use (s) but rather to use the plural form instead.